### ISSUE TITLE
[Feature Request]: API versions for Recovery Services Vaults and related resources are outdated

### ISSUE THREAD

**Issue by tyconsulting** [Azure/ResourceModules#1388]
### Describe the bug

The latest API version for `icrosoft.RecoveryServices/vaults` and its child resources such as `Microsoft.RecoveryServices/vaults/backupconfig` and `Microsoft.RecoveryServices/vaults/backupPolicies` are outdated. at the moment, the latest API version is `2022-02-01`

### To reproduce

N/A

### Code snippet

```Bicep
https://docs.microsoft.com/en-us/azure/templates/microsoft.recoveryservices/vaults?tabs=bicep
```

**Comment by MrMCake** [comment]
@prasanjeets are you already working on this? The RSV test will fail eventually - but we can of course update it ahead of time (ideally with the new properties).

**Comment by eriqua** [comment]
Changes have been implemented through PR #1572. Closing.
